### PR TITLE
live555: Build and install improvements

### PR DESCRIPTION
**Pull request by Barrett17**
* Fix install directories.
* Add BUILD_REQUIRES.
* Add REQUIRES_DEVEL.

### CODE REVIEW

File: media-video/live555/live555-2016.01.29.recipe [hunk 1]
```diff
@@ -23,10 +23,18 @@ REQUIRES="
 	haiku$secondaryArchSuffix
 	"
 
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
```
Comment (korli) [inline media-video/live555/live555-2016.01.29.recipe]: This comes after REQUIRES_DEVEL

File: media-video/live555/live555-2016.01.29.recipe [hunk 2]
```diff
@@ -23,10 +23,18 @@ REQUIRES="
...
 PROVIDES_devel="
 	live555${secondaryArchSuffix}_devel = $portVersion
 	"
 
+REQUIRES_devel="
+	live555$secondaryArchSuffix == $portVersion base
+"
+
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```
Comment (korli) [inline media-video/live555/live555-2016.01.29.recipe]: To be removed if present in BUILD_REQUIRES

Comment (Barrett17) [inline media-video/live555/live555-2016.01.29.recipe]: I have to move the REQUIRES_DEVEL content in BUILD_REQUIRES?

Comment (korli) [inline media-video/live555/live555-2016.01.29.recipe]: This comment goes with the line 39 of the patched recipe. You added  BUILD_REQUIRES, with one identical line in BUILD_PREREQUIRES. I suggest to remove the latter.

Comment (Barrett17) [inline media-video/live555/live555-2016.01.29.recipe]: Ok, I see thanks for clarifying.

**Pull request merged**